### ISSUE TITLE
Don't attempt to read dotfiles

### ISSUE THREAD

**Issue by RichardLitt** [HaroldMills/Vesper#184]
While trying to import recordings, I noted that Vesper tried to read dot files created automatically by OSX. 

Here is what those look like:

```
$ sudo ls
._Calls.yaml	._Coarse.yaml	Calls.yaml	Coarse.yaml
```

I had to manually delete them to remove the warnings, which I don't think were critical. Still, Vesper could ignore files starting with `._`.

**Comment by HaroldMills** [comment]
I'll look into that. Thanks!

**Comment by HaroldMills** [comment]
This is fixed in 7526741212c4d3dd7f94ebec259abbdbd7badaaa. Vesper now ignores preset and audio files whose names start with a dot.

**Comment by RichardLitt** [comment]
Thank you!
